plugins/hsqldb-adapter/src/main/java/org/polypheny/db/hsqldb/stores/HsqldbSqlDialect.java

```diff
@@ -99,19 +99,7 @@ public SqlNode getCastSpec( AlgDataType type ) {
                 // We need to flag the type with a underscore to flag the type (the underscore is removed in the unparse method)
                 castSpec = "_BLOB";
                 break;
-            case INTERVAL_YEAR_MONTH:
-            case INTERVAL_DAY:
-            case INTERVAL_DAY_HOUR:
-            case INTERVAL_DAY_MINUTE:
-            case INTERVAL_DAY_SECOND:
-            case INTERVAL_HOUR_MINUTE:
-            case INTERVAL_HOUR:
-            case INTERVAL_HOUR_SECOND:
-            case INTERVAL_MINUTE:
-            case INTERVAL_MONTH:
-            case INTERVAL_SECOND:
-            case INTERVAL_MINUTE_SECOND:
-            case INTERVAL_YEAR:
+            case INTERVAL:
                 castSpec = "INTERVAL";
```

Comment:
See comment on line 100, should it not be `"_INTERVAL"`

Reply:
Adjusted